### PR TITLE
azure test script: make report at the begining

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: scripts/runalltests.sh [hunk 1]
```diff
@@ -27,13 +27,20 @@ cd test
...
-exec &> /dev/null
-#cd $workdir
-#rm results.zip
-#zip results.zip *.txt
-date=$(date +%Y-%m-%d)
+#Prepare Header of the report
+exec &> $workdir/testreport.txt
+exectime=$(echo "$timeend - $timestart" | bc)
+echo "REPORT"
+echo "Test execution time: "$exectime
```
Comment (chriseth) [inline scripts/runalltests.sh]: Can you change this to `"Test execution time: $exectime s"`?

File: scripts/runalltests.sh [hunk 2]
```diff
@@ -27,13 +27,20 @@ cd test
...
+cat $workdir/testerror.txt
+cat $workdir/testreport.txt | cat - $workdir/testlog.txt > temp && mv temp $workdir/testlog.txt
```
Comment (chriseth) [inline scripts/runalltests.sh]: Wouldn't `cat $workdir/testreport.txt $workdir/testlog.txt > $workdir/testlog.txt` do the same?

Comment (winsvega) [inline scripts/runalltests.sh]: cat: input file is output file

Comment (chriseth) [inline scripts/runalltests.sh]: Ah ok, but you can just use a new file.

**Pull request merged**